### PR TITLE
Des 89 refactor folders structure

### PR DESCRIPTION
**Pull request by Daniielmayo**

### CODE REVIEW

File: src/auth/registerPageNavbar/ResgisterPageNavbar.jsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import { useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import Swal from 'sweetalert2';
-import { Header } from '../../components/organims/header/navbar/Header';
+
```
Comment (DanielCalleCO) [inline src/auth/registerPageNavbar/ResgisterPageNavbar.jsx:4]: could you please delete this line?

File: src/auth/registerPageNavbar/ResgisterPageNavbar.jsx [hunk 2]
```diff
@@ -107,20 +108,20 @@ export const RegisterPageNavbar = () => {
...
           </div>
 
-          <Link className="register-go_login" to="/auth/login">
+          <Link className="redirect" to="/auth/login">
             ¿Ya tienes cuenta?
           </Link>
 
-          <div className="container-btn_create">
+          <div className="container-btn">
             {/* <Link className='ingresar-form' to='/homeview'> <span  className='crear_span'>Crear cuenta</span> </Link> */}
```
Comment (DanielCalleCO) [inline src/auth/registerPageNavbar/ResgisterPageNavbar.jsx:123]: what functionality does this comment have?

File: src/auth/sesionPageNabvar/SesionPageNavbar.jsx [hunk 3]
```diff
@@ -1,11 +1,12 @@
 import { useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import Swal from 'sweetalert2';
-import { Header } from '../../components/organims/header/navbar/Header';
+
```
Comment (DanielCalleCO) [inline src/auth/sesionPageNabvar/SesionPageNavbar.jsx:4]: remove this line pls

File: src/components/organims/Dashboard/Dashborad.css
Comment (DanielCalleCO) [inline src/components/organims/Dashboard/Dashborad.css:1]: Change this file name to: Dashbo**ar**d.css

File: src/components/organims/Dashboard/Dashborad.jsx
Comment (DanielCalleCO) [inline src/components/organims/Dashboard/Dashborad.jsx:1]: DashboARd.jsx

File: src/components/organims/Path/Path.jsx [hunk 4]
```diff
@@ -1,10 +1,11 @@
 import { Link } from 'react-router-dom';
 import LockIcon from '@mui/icons-material/Lock';
-import './learningPath.css';
-import { Sidebar } from '../../../molecules/Sidebar/Sibebar';
-import { PrivateRoutes } from '../../../../models/routes';
 
-export const LearningPath = () => {
+import { PrivateRoutes } from '../../../models/routes';
+// import { Sidebar } from '../../molecules/Sidebar/Sidebar';
```
Comment (DanielCalleCO) [inline src/components/organims/Path/Path.jsx:5]: Do we need this import? If not, please remove it

File: src/components/organims/Path/Path.jsx [hunk 5]
```diff
@@ -1,10 +1,11 @@
...
+import './Path.css';
+
+export const Path = () => {
   return (
     <>
       {/* <div className='containerRoutes' >HomeViewRoutes</div> */}
```
Comment (DanielCalleCO) [inline src/components/organims/Path/Path.jsx:11]: same here, if we need this, keep it AND COMMENT ON IT why we need it, if not, please remove it

File: src/components/organims/Path/Path.jsx [hunk 6]
```diff
@@ -74,7 +75,7 @@ export const LearningPath = () => {
           </Link>
         </div>
       </div>
-      <Sidebar />
+      {/* <Sidebar /> */}
```
Comment (DanielCalleCO) [inline src/components/organims/Path/Path.jsx:78]: why is this commented?

File: src/components/organims/carousel/Carousel.jsx [hunk 7]
```diff
@@ -9,7 +9,7 @@ import './Carousel.css';
 
 const Carousel = () => {
   const slideshow = useRef(null);
-  // const intervalSlideshow = useRef(null);
+  const intervalSlideshow = useRef(null);
```
Comment (DanielCalleCO) [inline src/components/organims/carousel/Carousel.jsx:12]: what does interval slideshow mean? did you mean `slideshowInterval`?

File: src/components/organims/carousel/Carousel.jsx [hunk 8]
```diff
@@ -9,7 +9,7 @@ import './Carousel.css';
...
 
   const following = () => {
     //Comprobamos que slideshow tenga elementos
```
Comment (DanielCalleCO) [inline src/components/organims/carousel/Carousel.jsx:15]: Remove or translate and explain why we need it

**Review by DanielCalleCO — CHANGES_REQUESTED** [review]

File: src/auth/registerPageNavbar/ResgisterPageNavbar.jsx (hunk 2 above)
Comment (Daniielmayo) [reply src/auth/registerPageNavbar/ResgisterPageNavbar.jsx:123]: None have been deleted

File: src/auth/sesionPageNabvar/SesionPageNavbar.jsx (hunk 3 above)
Comment (Daniielmayo) [reply src/auth/sesionPageNabvar/SesionPageNavbar.jsx:4]: Well I could delete it but that line separates my imports which I have created and external packages

Comment (Daniielmayo) [reply src/auth/sesionPageNabvar/SesionPageNavbar.jsx:4]: I don't know if you think it's convenient to delete it

File: src/components/organims/Path/Path.jsx (hunk 4 above)
Comment (Daniielmayo) [reply src/components/organims/Path/Path.jsx:5]: Deleted

File: src/components/organims/Path/Path.jsx (hunk 5 above)
Comment (Daniielmayo) [reply src/components/organims/Path/Path.jsx:11]: not really needed it was already removed

**Review by DanielCalleCO — APPROVED** [review]
Awesome job! now LGTM

File: src/auth/sesionPageNabvar/SesionPageNavbar.jsx (hunk 3 above)
Comment (DanielCalleCO) [reply src/auth/sesionPageNabvar/SesionPageNavbar.jsx:4]: sorry, I did not see your comments 😅... is not a good practice to let the imports be divided. Everything should be in the same place and without spaces between it. You can order it to allow all the related imports to be together.

File: src/components/organims/Dashboard/Dashborad.css
Comment (DanielCalleCO) [reply src/components/organims/Dashboard/Dashborad.css:1]: did you change this file name finally? the name was wrong

File: src/components/organims/Dashboard/Dashborad.jsx
Comment (DanielCalleCO) [reply src/components/organims/Dashboard/Dashborad.jsx:1]: did you change this file name finally? the name was wrong

File: src/components/organims/Dashboard/Dashborad.css
Comment (Daniielmayo) [reply src/components/organims/Dashboard/Dashborad.css:1]: Ready

Comment (DanielCalleCO) [reply src/components/organims/Dashboard/Dashborad.css:1]: use "Done". Good job parce!

**Review by cristianarce89 — APPROVED** [review]

**Pull request merged**